templates/shared/runtime/bin/setup-local-disks

```diff
@@ -256,11 +260,21 @@ fi
 ## Get devices of NVMe instance storage ephemeral disks
 EPHEMERAL_DISKS=($(realpath "${disks[@]}" | sort -u))
 
+## Also bail early if there are not enough disks for raid10
+if [[ "${DISK_SETUP}" == "raid10" && "${#EPHEMERAL_DISKS[@]}" -lt 4 ]]; then
+  echo "raid10 requires at least 4 disks, but only ${#EPHEMERAL_DISKS[@]} found, skipping disk setup"
+  exit 0
```

Comment:
I don't think exiting successfully is the right behavior here. We should either exit with an error (because the user's config is not valid for the instance type) or fall back to one of the other DISK_SETUP values (because otherwise the instance store volumes won't be used at all).

Reply:
Yeah, I agree, "silent" failure isn't probably what we want. Fixed.